sklearn/metrics/_plot/tests/test_plot_roc_curve.py

```diff
@@ -38,7 +38,8 @@ def test_plot_roc_curve_error_non_binary(data):
      ("auto", "response methods not defined"),
      ("bad_method", "response_method must be 'predict_proba', "
                     "'decision_function' or 'auto'")])
-def test_plot_roc_curve_error_no_response(data_binary, response_method, msg):
+def test_plot_roc_curve_error_no_response(pyplot, data_binary, response_method,
+                                          msg):
```

Comment:
More accidental commits? :)

Reply:
I have been forgetting to branch from upstream/master recently. 😅